### PR TITLE
Black box bounded context tests

### PR DESCRIPTION
**Pull request by serhii-lekariev**
This PR updates most of the tests to use `BlackBoxBoundedContext` as opposed to previously used `TodoClient`.

### CODE REVIEW

**Comment by serhii-lekariev** [comment]
@dmdashenkov, PTAL.

File: api-java/src/test/java/io/spine/examples/todolist/c/procman/TaskCreationWizardTest.java [hunk 1]
```diff
@@ -474,65 +389,37 @@ protected void setUp() {
...
 
+    @SuppressWarnings({"PackageVisibleField" /* improves readability */, "WeakerAccess"})
     private abstract static class CommandTest {
 
-        private TaskCreationWizard wizard;
-
-        private MemoizingCommandHandler memoizingHandler;
+        BlackBoxBoundedContext boundedContext;
+        TaskId taskId;
+        TaskCreationId processId;
```
Comment (dmdashenkov) [inline api-java/src/test/java/io/spine/examples/todolist/c/procman/TaskCreationWizardTest.java]: Please use getters instead.

**Comment by serhii-lekariev** [comment]
@dmdashenkov, PTAL.

File: api-java/src/test/java/io/spine/examples/todolist/c/procman/TaskCreationWizardTest.java [hunk 2]
```diff
@@ -364,10 +364,10 @@ protected void setUp() {
         void testCancelProc() {
             CancelTaskCreation cmd = CancelTaskCreationVBuilder
                     .newBuilder()
-                    .setId(processId)
+                    .setId(processId())
                     .build();
-            boundedContext.receivesCommand(cmd)
-                          .assertEntity(TaskCreationWizard.class, processId)
+            boundedContext().receivesCommand(cmd)
+                          .assertEntity(TaskCreationWizard.class, processId())
```
Comment (dmdashenkov) [inline api-java/src/test/java/io/spine/examples/todolist/c/procman/TaskCreationWizardTest.java]: Please fix the code alignment here in this file.

**Pull request merged**